### PR TITLE
feat(grafana): Install and configure grafana plugin

### LINKED ISSUE #15: Plugin: Install and configure Grafana plugin
Provide a connection to Operate First grafana instance enabling Backstage to link all the dashboards from individual components.

https://github.com/K-Phoen/backstage-plugin-grafana/

### PR DESCRIPTION
**Pull request by SamoKopecky**
Resolves #15 
- Install and configure grafana plugin
- Update `README.md` with grafana plugin section
- Add  `GRAFANA_TOKEN` variable

### CODE REVIEW

File: package.json
Comment (tumido) [inline package.json:9]: Hm, can you please move the `BACKEND_SECRET` to the `.env` file as well?

I don't like the messy set+source+set, it's a shame backstage cli doesn't support dotenv... Can you please instead try installing `env-cmd` as a dev dependency to root of all workspaces and extend the `start-backend` script with it? We shouldn't need env variables on the frontend. https://github.com/toddbluhm/env-cmd

**Review by tumido — CHANGES_REQUESTED** [review]
I think README typos and repo pointers should be in a separate PR, it's unrelated to Grafana. Maybe the conversion of env variables to `.env` file should be a separate PR as well.. Let's keep PRs atomic if possible.

**Comment by tumido** [comment]
> Maybe the conversion of env variables to `.env` file should be a separate PR as well.. Let's keep PRs atomic if possible.

https://github.com/operate-first/service-catalog/pull/34 is precisely why that should be a separate PR. PR #34  is dependent on grafana plugin now, which shouldn't be the case.

File: README.md
Comment (SamoKopecky) [reply README.md:15]: Moved to another PR.

File: README.md
Comment (SamoKopecky) [reply README.md:23]: Moved to another PR.

File: README.md
Comment (SamoKopecky) [reply README.md:27]: Moved to another PR.

File: app-config.local.yaml
Comment (SamoKopecky) [reply app-config.local.yaml]: Moved to another PR.

File: package.json
Comment (SamoKopecky) [reply package.json]: Moved to another PR.

**Comment by SamoKopecky** [comment]
Waiting for #41 to merge so I can add a variable to `.env.example` file.
/hold

**Review by tumido — APPROVED** [review]
/lgtm
/approve

ready to merge once rebased

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**
